dockstore-yaml-proposals/kstars/docker-compose.yaml

```diff
@@ -13,8 +13,8 @@ services:
     deploy:
       resources:
         limits:
-          cpus: '8'
-          memory: 64000M
+          cpus: '4'
+          memory: 32000M
```

Comment:
I don't think this will work on Blackbalsam, that is the CPU/Memory that is allocated to each node.

Reply:
I copied the '8' and '6400M' from imagej -- maybe that is for the huge mouse brain images???

Reply:
In general the CPU/Memory settings should be set to the lowest possible and then increase if the performance is poor.

Reply:
'2' and '8000M' for the limits is probably more than enough for now.

Reply:
Changed the limits to 2 and 8000M